custom/opm/reports.py

```diff
@@ -888,7 +896,7 @@ def row_objects(self):
         self._debug_data = []
         for index, row in enumerate(self.get_rows(self.datespan), 1):
             try:
-                rows.append(self.get_row_data(row, index=index))
```

Comment:
What this meant was, instead of looking for `child1_attended_vhnd` it looked for `child23_attended_vhnd` (when `index` is 23)